### ISSUE TITLE
`wrong-attribute-order` lint is missing

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#632]
Consider this EO object:

```eo
# Foo.
[x] > foo
  [] > bar
    stdout "bar" > @
  eq. +> tests-foo
    foo "Jeff"
    "Foo"
  [] > tar
    stdout "tar" > @
```

Here, `foo` has three attributes inside: `bar`, `tar`, and `tests-foo` (test attribute). They should be sorted this way: first, object attributes go, then test attributes. In the example above, we should complain about `tests-foo` and `tar` positions.

This is the correct snippet:

```eo
# Foo.
[x] > foo
  [] > bar
    stdout "bar" > @
  [] > tar
    stdout "tar" > @
  eq. +> tests-foo
    foo "Jeff"
    "Foo"
```

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel make sense, let's do it

**Comment by 0crat** [comment]
@h1alexbel Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running balance is [+350](https://www.eolang.org/zerocracy/objectionary-vitals.html).

**Comment by h1alexbel** [comment]
resolved

**Comment by 0crat** [comment]
@h1alexbel Thanks for closing this issue! You've earned +4 points for this: +8 as a basis; -20 for 388 days of delay; +16 to give you at least something. Your running score is [+4](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
